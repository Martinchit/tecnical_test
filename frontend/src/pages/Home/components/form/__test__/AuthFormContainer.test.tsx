import React from 'react';
import { shallow } from 'enzyme';

import { AuthFormContainer } from '../AuthFormContainer';

describe('AuthFormContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AuthFormContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
