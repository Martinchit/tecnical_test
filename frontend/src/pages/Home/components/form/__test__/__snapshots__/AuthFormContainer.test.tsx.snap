// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthFormContainer renders correctly 1`] = `
<div
  className="sc-AxjAm exsDco"
/>
`;
